apps/backend/src/app/modules/admin/admin.service.ts

```diff
@@ -164,20 +186,25 @@ export class AdminService {
   ) {
     throwIfEmpty(update);
 
-    const user: any = await this.db.user.findUnique({
+    const user = await this.db.user.findUnique({
       where: { id: userID },
       include: { profile: true }
     });
 
     if (!user) throw new NotFoundException('User not found');
 
+    const activities = [AdminActivityType.USER_UPDATE];
+
```

Comment:
Is there much point to store this general activity type, when we have all the fine-grained stuff?